blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.jsx

```diff
@@ -25,44 +27,46 @@ const RightRailAdvancedLayout = ({ children }) => {
 		rightRailBottom,
 		fullWidth2,
 		footer,
-	] = children;
+	] = React.Children.toArray(children);
 	const featureList = useFeatureList();
 
 	return (
-		<>
-			<header className="page-header">{navigation}</header>
-			<section role="main" id="main" className="main" tabIndex="-1">
-				<div className="container layout-section">
-					<div className="row">
-						<div className="col-sm-xl-12 layout-section wrap-bottom">{fullwidth1}</div>
-					</div>
-					<div className="row advanced-grid">
-						<section className="col-sm-md-12 col-lg-xl-8 advanced-grid-desktop-main-area">
-							<RenderChild Item={main} tabletPlacement="2" />
-							<RenderChild Item={main2} tabletPlacement="4" />
-						</section>
-						<section className="col-sm-md-12 col-lg-xl-4 layout-section">
-							<RenderChild Item={rightRailTop} tabletPlacement="1" />
-							<RenderChild Item={rightRailMiddle} tabletPlacement="3" />
-							<RenderChild Item={rightRailBottom} tabletPlacement="5" />
-						</section>
-					</div>
-					{featureList["7"] > 0 && (
-						<div className="row">
-							<div className="col-sm-xl-12 layout-section wrap-bottom">{fullWidth2}</div>
-						</div>
-					)}
-				</div>
+		<div className={LAYOUT_CLASS_NAME}>
+			{navigation ? (
+				<Stack role="banner" className={`${LAYOUT_CLASS_NAME}__navigation`}>
```

Comment:
It's too bad we're not still using a semantic `<header>` here, but you've got ARIA covered by having `role="banner"`. 👍🏼 
Wondering if `<Stack>` could/should have a `tagName` attribute for cases like these, rather than getting the default `<div>` container. 🤔 
https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.00/src/components/stack/index.jsx#L18
(thoughts of a different scope)

Reply:
That's a good question. I think for the scope of this requirement it is not needed.

Reply:
Why wouldn't we update the `Stack` component to do this? It's a nice addition and keeps our code cleaner

Reply:
See PR in Components for updated `Stack` - https://github.com/WPMedia/arc-themes-components/pull/94

Reply:
I'm not sure I understand why this is necessary. Is this an accessibility suggestion? @matthewroach

Reply:
We should use semantic elements over additional attributes to achieve the same outcome. The first rule of ARIA is not to use ARIA. Also we should be setting a standard here, and I worry having the `role` here would also lead to confusion in the future